includes/src/Report/Accordians.js

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import Collapse from './Collapse'
+
+class Accordians extends React.Component {
+
+    getRank(pluginDataList) {
+        var testCaseMax = 0;
+        var maxUserRank = -1;
+        var maxOWTFRank = -1;
+        for (var i = 0; i < pluginDataList.length; i++) {
+            if (pluginDataList[i].user_rank != null && pluginDataList[i].user_rank != -1) {
+                if (pluginDataList[i].user_rank > maxUserRank) {
+                    maxUserRank = pluginDataList[i].user_rank;
+                }
+            } else if (pluginDataList[i].owtf_rank != null && pluginDataList[i].owtf_rank != -1) {
+                if (pluginDataList[i].owtf_rank > maxOWTFRank) {
+                    maxOWTFRank = pluginDataList[i].owtf_rank;
+                }
+            }
+        }
+        testCaseMax = (maxUserRank > maxOWTFRank)
```

Comment:
@delta24  Actually I was using https://orktes.github.io/atom-react/ for auto indentation and stuff. This conditional multi-lined by this module. Just check once if you fine with this module.